internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -5933,6 +5933,80 @@ func TestHandleTSpendVotes(t *testing.T) {
 	}})
 }
 
+func TestHandleVerifyMessage(t *testing.T) {
+	t.Parallel()
+
+	msg := "test message"
+	nonCompactSig := "MEUCIQD8wKh2jPvO/PLK3Xz7D7GO0I3S4q6EvvGkdKPTUbJvAwIgD" +
```

Comment:
I would very much prefer that this test data include the details needed to reconstruct it.  Namely, the private key involved.  

However, from the looks of it, this is probably from a legitimate mainnet address which you don't want to the share the private key for.  Thus, I suggest using one of the standard private keys that are used elsewhere (namely `0x00....1`) and documenting it here in a comment as is done elsewhere in crypto-related code.

Next up, what is a `nonCompactSig`?  It doesn't seem to line up with the claim.  Actually, most of the this doesn't match reality.  The address claimed to be `uncompressedPkAddr` isn't one, since that would start with `Dk` if it were true.

Reply:
For reference, here the relevant values:

```
private key: 0x0000000000000000000000000000000000000000000000000000000000000001
message: test message
compressed pkh addr: DsmcYVbP1Nmag2H4AS17UTvmWXmGeA7nLDx
uncompressed pkh addr: DsbnCMAYV13buumdjHuwiJeJWZWvgjZRTbE
compressed compact sig: H18ier4CIfSBOk0FKPjO4mggno0ES1w2P+41GpJnnyiSRWdE2n02YwE29Sw0n2ALT3M1Q1+GQW7moKqsem1COF8=
uncompressed compact sig: G18ier4CIfSBOk0FKPjO4mggno0ES1w2P+41GpJnnyiSRWdE2n02YwE29Sw0n2ALT3M1Q1+GQW7moKqsem1COF8=
```

Reply:
`nonCompactSig` is referring to a signature that is greater than the expected compact signature size. Perhaps `nonCompactSigSize` is a better name?

Reply:
going with `invalidCompactSig`

Reply:
That's better, but I would call it `malformedCompactSig` then.  In terms of signatures, the word invalid is typically used to refer to signature that is properly formed but just is not correct for the given message and address pair.  Thus `invalidCompactSig` is also a useful test to have, but it should be a signature for the wrong message,meaning it validates for the other message, but not the one that is being checked.

Reply:
gotcha, updating.